.github/workflows/model_ci.yml

```diff
@@ -40,21 +40,21 @@ jobs:
         rasa test core --stories test_stories/stories.md --fail-on-prediction-errors
 
     - name: Cross-validate NLU model
-      if: github.event_name == 'push' && github.ref != 'refs/heads/master'
+      if: github.event_name == 'pull_request'
```

Comment:
does this mean that if I push a new commit to an existing PR it won't run again?

Reply:
it will. the `synchronize` type for `pull_request` will trigger then (`synchronize` is not a really an intuitive name for what it does 😄 )